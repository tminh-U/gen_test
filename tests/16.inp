8
11 5 18 5 10 5 3 4 
10
6 6
8 8
3 7
3 6
6 7
3 5
8 8
2 2
8 8
8 8
