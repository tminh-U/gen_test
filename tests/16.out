2
2
8
7
3
5
2
2
2
2
